packages/flutter_tools/test/android/android_sdk_test.dart

```diff
@@ -132,9 +132,11 @@ void main() {
 
           final AndroidSdk sdk = AndroidSdk.locateAndroidSdk();
           expect(sdk.directory, realSdkDir);
-          expect(sdk.ndkDirectory, realNdkDir);
-          expect(sdk.ndkCompiler, realNdkCompiler);
-          expect(sdk.ndkCompilerArgs, <String>['--sysroot', realNdkSysroot]);
+          print(AndroidNdk.explainMissingNdk(sdk.directory));
```

Comment:
I'm assuming this `print` was unintentional. I've removed it in https://github.com/flutter/flutter/pull/17401/files; please let me know if that was a mistake.

Reply:
Thanks, @Hixie. That was a debugging print left-over.